content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -17,7 +18,7 @@ const ActivityFeedItem = React.createClass({
           <a className="feed-link" href={site.url} ref="link">{prettyUrl(site.url)}</a>
         </div>
         <div className="feed-stats">
-          <div>1:26pm</div>
+          <div>{site.lastVisitDate && moment(site.lastVisitDate / 1000).format("h:mma")}</div>
```

Comment:
Funny, I just answered my own question with the [screenshot below](https://github.com/mozilla/activity-streams/pull/197#issuecomment-188397205). It looks like the two "Top Activity" links don't have last visited dates, so they just show the "..." menu icon.